.env.example

```
@@ -0,0 +1,56 @@
+JWT_ACCESS_TOKEN_EXPIRY=3600000
+JWT_REFRESH_TOKEN_EXPIRY=604800000
+
+# Spring AI - OpenAI Configuration
+OPENAI_API_KEY=your-openai-api-key-here
+OPENAI_MODEL=gpt-4o-mini
+OPENAI_TEMPERATURE=0.7
+
+# Spring AI - Google Gemini Configuration
+GOOGLE_API_KEY=your-google-api-key-here
+GOOGLE_CLOUD_PROJECT_ID=your-google-cloud-project-id
+GOOGLE_CLOUD_LOCATION=asia-northeast3
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GOOGLE_CLOUD_LOCATION key should go before the GOOGLE_CLOUD_PROJECT_ID key

(UnorderedKey)